tes-adapter/build.gradle

```diff
@@ -1,16 +1,37 @@
-plugins {
-    id 'java'
-}
+group 'com.epam.pipeline.adapter'
+version '1.0-SNAPSHOT'
 
-group 'com.epam.pipeline'
+buildscript {
+    ext {
+        springBootVersion = '1.5.2.RELEASE'
+    }
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+    }
+}
 
-sourceCompatibility = 1.8
+apply plugin: 'java'
+apply plugin: 'org.springframework.boot'
 
 repositories {
     mavenCentral()
 }
 
+jar {
+    manifest {
+        attributes 'Main-Class': 'com.epam.pipeline.adapter.TesAdapterApplication'
```

Comment:
here we have com.epam.pipeline.adapt**E**r.TesAdapterApplication but package adapt**O**r

Reply:
fixed